### DISCUSSION TITLE
Can cFS be distributed as part of a disk image for a SBC?

### DISCUSSION THREAD

**Discussion by trholding** [nasa/cFS#406] [Q&A]
We are developing a low cost Single Board computer for makers and students who are also interested in spacecraft.

I have some questions as to what are the dos and dont's when using cFS.

- Is distributing cFS as part of a downloadable disk image legal?
- Can cFS apps be distributed or would that be regulated by ITAR?
- We hope to write example guidance & navigation apps for cFS for pico & cube sats so that makers/students can use it as a starting point or reference.
- We also hope to use cFS for IoT, Robotics, Hobby Electronics applications not related to spacecraft. 
- We do not charge money for the software.
- We are not US based.

Any pointers are appreciated.

--Vulcan

**Comment by skliper** [comment]
https://github.com/nasa/cFS#other-cfs-related-elementstoolsappsdistributions contains a decent list to start from but we haven't really gotten any community input.  I suggest taking a close look at https://github.com/OpenSatKit/OpenSatKit in particular.

I can't provide legal/ITAR advice, but the majority of the cFS open source repositories are Apache 2.0 or NOSA licensed (and a few apps are expected to transition from NOSA to Apache 2.0 with their next release).  There are no additional restrictions other than compliance with the applicable license (see each repo for details).  There is a thread on the community email list that contains the app list for pending releases at https://lists.nasa.gov/archives/list/cfs-community@lists.nasa.gov/thread/B6ZBEVDTWSXBX2HXJHEDGVDB44EK7HIK/.

We definitely encourage collaboration and look forward to following your progress!
